awslambda/deletesnippet/index.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+console.log('Loading function');
+
+const aws = require('aws-sdk');
+
+const s3 = new aws.S3({ apiVersion: '2006-03-01' });
+
+
+exports.handler = (event, context, callback) => {
+    let userId = event.pathParameters.user_id;
```

Comment:
If there is no assignment to this variable after this point, you can make this a const.